packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -240,24 +256,43 @@ export default function MainFeedLayout({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [sortingEnabled, selectedAlgo, loadedSettings, loadedAlgo]);
 
-  const FeedPageComponent = shouldUseMobileFeedLayout
+  const MobileOrDesktopLayout = shouldUseMobileFeedLayout
     ? FeedPageLayoutMobile
     : FeedPage;
+  const FeedPageComponent = shouldUseCommentFeedLayout
+    ? CommentFeedPage
+    : MobileOrDesktopLayout;
 
-  const disableTopPadding = isFinder || shouldUseMobileFeedLayout;
+  const disableTopPadding =
+    isFinder || shouldUseMobileFeedLayout || shouldUseCommentFeedLayout;
 
   return (
     <FeedPageComponent
       className={classNames('relative', disableTopPadding && '!pt-0')}
     >
       {isSearchOn && search}
-      {feedProps && (
-        <Feed
-          {...feedProps}
-          className={classNames(
-            shouldUseMobileFeedLayout && !isFinder && 'laptop:px-6',
-          )}
+      {shouldUseCommentFeedLayout ? (
+        <CommentFeed
+          feedQueryKey={generateQueryKey(RequestKey.CommentFeed, null)}
+          query={COMMENT_FEED_QUERY}
+          analyticsOrigin={Origin.CommentFeed}
+          emptyScreen={
+            <ProfileEmptyScreen
+              title="Nobody has replied to any post yet"
+              text="You could be the first you know?"
+            />
+          }
+          commentClassName={commentClassName}
         />
+      ) : (
+        feedProps && (
+          <Feed
+            {...feedProps}
+            className={classNames(
+              shouldUseMobileFeedLayout && !isFinder && 'laptop:px-6',
+            )}
+          />
+        )
```

Comment:
More for my understanding, I am not sure I understand why this change is needed if the comment feed is just added to the user replies page?

Reply:
It also replaces `Most discussed feed` for users in the experiment basically.